style-guide/latin-abbr.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
Probably want to include poorly-punctuated versions of these, such as `eg.`, `ie.` (with and without trailing periods, surrounded by spaces or surrounded by asterisks (italics):

```
Some people, *ie* people who don't read the style guide, might put Latin abbreviations in italics. *Eg* like this.
```